sql/_35_fig_cake/grant_revoke_redefine/cases/cbrd_25607_04.sql

```diff
@@ -0,0 +1,136 @@
+/* Verified the CBRD-25607
+As-is: When grant/revoke from dba or owner group user, the grantor_name is set to the grantor name
+To-be: When grant/revoke from dba or owner group user, the grantor_name is set to the owner name
+ */
+
+
+select 'create test users' from dual;
+CREATE USER owner;
+CREATE USER grantable;
+CREATE USER dba_member groups dba;
+CREATE USER owner_member groups owner;
+CREATE USER grantable_member groups grantable;
+CREATE USER temp_user;
+
+
+evaluate 'create owner.tbl, owner.hello';
+CREATE TABLE owner.tbl (col1 int);
+
+CREATE FUNCTION owner.hello(i varchar) RETURN STRING
+AS LANGUAGE JAVA
+NAME 'SpTest.Hello(String) return java.lang.String';
+
+
+
+
+evaluate 'case 6: dba grants privileges to dba groups';
+evaluate 'connect to dba & owner.tbl, owner.hello grant to dba_member';
+CALL LOGIN('dba','') ON  CLASS db_user;
+GRANT select ON owner.tbl TO dba_member;
+GRANT EXECUTE ON PROCEDURE owner.hello TO dba_member;
+
+select grantor_name, grantee_name, object_name, auth_type, is_grantable from db_auth where grantee_name != 'PUBLIC' order by object_name, auth_type;
+
+evaluate 'connect to dba_member & owner.tbl, owner.hello grant to dba';
+CALL LOGIN('dba_member','') ON  CLASS db_user;
+GRANT select ON owner.tbl TO dba;
+GRANT EXECUTE ON PROCEDURE owner.hello TO dba;
+
+select grantor_name, grantee_name, object_name, auth_type, is_grantable from db_auth where grantee_name != 'PUBLIC' order by object_name, auth_type;
```

Comment:
```suggestion
select grantor_name, grantee_name, object_name, auth_type, is_grantable from db_auth where grantee_name != 'PUBLIC' order by grantee_name,object_name, auth_type;
```
object_name, auth_type 의 각 값이 동일한 것이 있기 때문에 변경하는 것이 좋겠습니다.

Reply:
반영했습니다.
리뷰 감사합니다.